main.js

```diff
@@ -5,17 +5,41 @@ const JSONStorage = require('node-localstorage').JSONStorage;
 const APPVERSION = require('./package.json').version;
 const https = require('https');
 const compareVersions = require('compare-versions');
+const minimist = require('minimist');
 
 const { app, BrowserWindow, ipcMain: ipc, Menu: menu, globalShortcut: gsc, shell } = electron;
 
+// slice args array differently
+// if calling electron directly in debug
+let arg_slice = 1
+
 if (process.env.NODE_ENV === 'development') {
   require('electron-debug')(); // eslint-disable-line global-require
+  args_slice = 2
+}
+
+let argv = minimist(process.argv.slice(arg_slice));
```

Comment:
Can you make this a const with a ternary operator? That gets rid of `arg_slice` completely and `argv` doesn't change anyway. Like this:

`const argv = minimist(process.argv.slice(process.env.NODE_ENV === 'development' ? 2 : 1));`